src/Files.App/Data/Contexts/ContentPage/ContentPageContext.cs

```diff
@@ -62,6 +62,20 @@ public ContentPageContext()
 			Update();
 		}
 
+		public void UpdateOpenTabsPreferences()
```

Comment:
Should this method really be under the page context? It seems it might be better as a helper.

Reply:
Is one between `LayoutPreferencesManager` (as a static method), `UIHelpers` fine or should I create a new `Helper` class?

Reply:
Let's go with a new class